### PR TITLE
Fix reference to defined format

### PR DESCRIPTION
**Pull request by stevedlawrence**

### CODE REVIEW

File: helloWorld/src/main/resources/helloWorld.dfdl.xsd [hunk 1]
```diff
@@ -12,7 +12,7 @@
   <xs:annotation>
     <xs:appinfo source="http://www.ogf.org/dfdl/">
 
-      <dfdl:format ref="GeneralFormat"
+      <dfdl:format ref="tns:GeneralFormat"
```
Comment (mbeckerle) [inline helloWorld/src/main/resources/helloWorld.dfdl.xsd:15]: We should get rid of this tns usage. Change prefix to hw?

File: hexWords/src/main/resources/hexWords.dfdl.xsd [hunk 2]
```diff
@@ -12,7 +12,7 @@
   <xs:annotation>
     <xs:appinfo source="http://www.ogf.org/dfdl/">
 
-      <dfdl:format ref="GeneralFormat"
+      <dfdl:format ref="tns:GeneralFormat"
```
Comment (mbeckerle) [inline hexWords/src/main/resources/hexWords.dfdl.xsd:15]: Here also. Suggest prefix is not "hw" though. How about "hxw"?

**Review by mbeckerle — APPROVED** [review]
+1 but I suggest also take this opportunity to remove the "tns" prefix definition. That's a convention that has been fairly unhelpful.

File: helloWorld/src/main/resources/helloWorld.dfdl.xsd (hunk 1 above)
Comment (stevedlawrence) [reply helloWorld/src/main/resources/helloWorld.dfdl.xsd:15]: The use of `tns` actually shows up in 22 places in 8 different files in this repos. I think I'd rather do it as a separate PR.

Note that it looks like a couple dozen of of schema repos still use `xmlns:tns=`. Most of them it's just in a couple files. We might want a script that goes around and updates all of them.

**Pull request merged**